style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Prefer `===` and `!===` to `==` and `!=`.
+* Avoid using arrow functions when not necessary.
```

Comment:
Not a technical problem, but I like being able to visually distinguish the two.

@christoomey That is an edge case to the rule and I would definitely prefer to see that. 

Any ideas on how we can word this and make sense?